src/genai/schema/_api.py

```diff
@@ -242,6 +251,14 @@ class RequestApiVersion(str, Enum):
     V2 = "v2"
 
 
+class RequestChatConversationIdRetrieveResultsRequest(ApiBaseModel):
```

Comment:
should we name it `AnyDictionary` and unify it as before?
(would require extra aliases I believe)

Reply:
Why? We just remove it in the next major version.